addons/mail/static/src/messaging/component/chatter_topbar/chatter_topbar.scss

```diff
@@ -13,6 +13,14 @@
     flex: 1;
     flex-direction: row;
     flex-wrap: wrap;
+
+    &.o-composer-open {
+        border-bottom: 1px solid gray('300');
+    }
+}
+
+.o_ChatterTopbar_button {
+    margin-bottom: -1px; /* Needed to allow "overriding" of the bottom border */
```

Comment:
Not ideal, can't we just "unset" the border for the currently active/hovered topbar element? Like keep width to prevent flicker but remove color?

Reply:
No we can't because the border is not on the topbar elements but on the topbar itself.
I took the master logic back even if it sucks

Reply:
hard to do much better, unfortunately

Reply:
Ok then, I guess we can keep it until we have a better idea.

Reply:
Should at least be `$border-width` then.